client/my-sites/domains/domain-management/transfer/transfer-page/index.tsx

```diff
@@ -443,10 +445,15 @@ const TransferPage = ( props: TransferPageProps ) => {
 							<br />
 							{ createInterpolateElement(
 								__(
-									'However, transferring a domain to another provider can take five to seven days during which no changes to the domain can be made. Read <a>this important information</a> before starting a transfer.'
+									'However, transferring a domain to another provider can take five to seven days during which no changes to the domain can be made. Read <supportLink>this important information</supportLink> before starting a transfer.'
```

Comment:
Changing the translation string will require a new translation. We can consider adding the label `[Status] String Freeze` and waiting for the translations to be completed before deploying, or using the original translations with `<a>`.

PCYsg-3hI-p2